src/sys/socket/sockopt.rs

```diff
@@ -478,6 +492,53 @@ unsafe impl<'a> Set<'a, usize> for SetUsize {
     }
 }
 
+/// Getter for a `OsString` value.
+struct GetOsString<T: AsRef<[u8]>> {
+    len: socklen_t,
+    val: T,
+}
+
+unsafe impl<T: AsRef<[u8]>> Get<OsString> for GetOsString<T> {
```

Comment:
You can remove the `unsafe` keyword on the `impl` block, because you use it on each individual method.

Reply:
The same